Cargo.toml

```diff
@@ -110,7 +110,8 @@ smolset = "^1.3.1"
 sshkeys = "^0.3.1"
 
 tide = "^0.16.0"
-tide-compress = "0.10.6"
+# Including brotli *very* slow, so don't do that. Including the "default" feature pulls a mime-type list from the internet on build, which isn't used.
+tide-compress = { version="0.10.6", default-features = false, features = [ "deflate", "gzip", "regex-check" ] }
```

Comment:
Isn't deflate just a shit gzip? Any reason to not just go gzip only?

Reply:
Nope, seems legit - [gzip is supported everywhere](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding#compressing_with_gzip)